src/main/java/net/snowflake/client/core/Event.java

```diff
@@ -5,12 +5,6 @@
 package net.snowflake.client.core;
 
 import com.google.common.base.Preconditions;
-import java.io.File;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.OutputStream;
-import java.io.PrintWriter;
-import java.util.zip.GZIPOutputStream;
```

Comment:
can you check the history of this changes and ensure this never used?

Reply:
The file was written in 2017 by @sfc-gh-hyu and appears unchanged since then (except for formatting). The commit message is "initial public version."